WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -80,10 +82,26 @@ final class StoreStatsV4PeriodViewController: UIViewController {
 
     private var isInitialLoad: Bool = true  // Used in trackChangedTabIfNeeded()
 
+    // To check whether the tab is showing the visitors and conversion views as redacted for custom range.
+    // This redaction is only shown on Custom Range tab with WordPress.com or Jetpack connected sites,
+    // while Jetpack CP sites has its own redacted for Jetpack state, and non-Jetpack sites simply has them empty.
+    private var isRedactedForCustomRangeShown: Bool {
+        guard timeRange.isCustomTimeRange,
+              let site = stores.sessionManager.defaultSite,
+              site.isJetpackConnected,
+              site.isJetpackThePluginInstalled else {
+            return false
+        }
+        return true
+    }
```

Comment:
Great work reusing this check! 👏

Reply:
This is nit-picking but I'd suggest renaming this variable to `unavailableVisitStatsDueToCustomRange` to make it clearer.

Reply:
Also: if want to follow Hicham's idea here [p1709800424149309/1709727324.356419-slack-C03L1NF1EA3], we'll have to check the range length to not redact the site visit stats for 1-day range.